src/RA_Dlg_AchEditor.cpp

```diff
@@ -118,8 +115,7 @@ void Dlg_AchievementEditor::SetupColumns(HWND hList)
     {
         col.mask = LVCF_TEXT | LVCF_WIDTH | LVCF_SUBITEM | LVCF_FMT;
         col.cx = COLUMN_WIDTH[i];
-        tstring colTitle = COLUMN_TITLE[i];	//	Take non-const copy
-        col.pszText = const_cast<LPTSTR>(colTitle.c_str());
+        col.pszText = NativeStr(COLUMN_TITLE[i]).data();
```

Comment:
In C++17 as long as the string is not a reference to const std::string::data gives you a non-const c-string.